### DISCUSSION TITLE
API: live data: small json request

### DISCUSSION THREAD

**Discussion by 2000ede** [tbnobody/OpenDTU#1713] [Ideas]
Hello,

I like the simple API /api/livedata/status. Exellent!

I use this with Google Sheets and create time series with selected values.

Now that another inverter has been connected, the json output is enlarged accordingly.

I don't know exactly, but apparently this has made Google Sheets less reliable. Every now and then I don't get any values.

Is it possible to implement another API command that only outputs the basic data?

I could make a corresponding suggestion!

**Comment by MetaChuh** [comment]
@2000ede 
there are api changes in the work right now,
which coincidentally might (or might not) solve your issue:

the future api will allow you to get data of a single inverter, instead of all at once.
so basically i'ts the same data amount, as before, when you only had a single inverter, and you can choose whch one to poll, if you have more than one.

the corresponding endpoint for test versions is currently `/api/livedata/status?inv=<inverter_serial_number>`

please see: [Announcements/Breaking API Changes](https://github.com/tbnobody/OpenDTU/discussions/1463) for all future changes

**Reply by 2000ede** [reply]
@MetaChuh thanks for the fast answer.

Since opendtu crashes more often now, I don't know exactly where the error is. This morning the API export ran great again.
Nothing is working right now...

Either way, I would include small API calls that contain the main values ​​of both WRs. Otherwise I have to do two in a row. That might put too much strain on the DTU.

A small API with
- Data age
- Producing status
- Power (all)
- Power (1)
- Power (2)
- Limit (1)
- Limit (2)
- Yield total (1)
- Yield total (2)

would be gread!

**Reply by MetaChuh** [reply]
> Nothing is working right now...

please define `Nothing is working right now`
does it mean, all inverters are offline, and the dtu does not power up, and you are not able to access the dtu web interface, and your  wireless network is not reachable from any device ?

**Reply by 2000ede** [reply]
I apologize for my unclear description.

I don't know where the "mistake" lies. Somehow the system is not stable.

-OpenDTU with two hoymiles inverters.
- Smart Micro Solar draws data and sets zero feed-in limit
- Tasmota ESP32 fetches values ​​every 3 seconds
- Tasmota ESP32 fetches data every minute
- Google Sheets fetches data every 15 minutes

It sounds like a lot, but it shouldn't be. But I will also reduce the access.

Google Sheets logs the data. Here you can see from time to time that he can't get his hands on data. Yesterday the DTU was completely frozen.

So there are many possible causes. I continue researching.

But I am of the opinion that small API calls are always good.

**Reply by 2000ede** [reply]
I also think a small API output for the so-called zero feed is very important.
The value is queried at very short intervals and the limit is then sent. Several times per minute.

**Reply by 2000ede** [reply]
@tbnobody for info
